app/src/main/resources/application.properties

```diff
@@ -26,7 +26,7 @@ registry.date=${timestamp}
 # Log
 %prod.quarkus.log.level=${LOG_LEVEL:INFO}
 %prod.quarkus.log.console.enable=true
-%prod.quarkus.log.category."io.apicurio".level=${REGISTRY_LOG_LEVEL:INFO}
+%prod.quarkus.log.category."io.apicurio".level=${LOG_LEVEL:INFO}
```

Comment:
Note to @Apicurio/developers - I really want to be able to control global vs. apicurio log levels separately.  Let's add another option in the operator if there is a gap, rather than make this change.